content/open-source/blend.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Blend
+sub: Sass support for LCH, Lab, and more CSS color spaces
+date: 2020-04-28
+image:
+  src: projects/blend.jpg
+oss: owner
+author:
+  - miriam
+  - oddbird
+index: Blend
+links:
+  docs: /blend/docs/
+  source: https://github.com/oddbird/blend
+badges:
+  - name: npm
+    src: https://badge.fury.io/js/%40oddbird%2Fblend.svg
+    url: https://www.npmjs.com/package/@oddbird/blend
```

Comment:
I note that this isn't actually being used/displayed, but 🤷

Reply:
@jgerigmeyer it is now, if you want to review that